routes/courses.js

```diff
@@ -129,6 +129,27 @@ router.delete('/:id', async function (req, res, next) {
   }
 });
 
+/*
+ * GET /courses/{id}/students
+ * Returns a list containing the User IDs of all
+ * students currently enrolled in the Course. 
+ * Only an authenticated User with 'admin' role or an 
+ * authenticated 'instructor' User whose ID matches the 
+ * instructorId of the Course can fetch the list of 
+ * enrolled students.
+ *
+ */
+router.get('/:id/students', async function (req, res, next) {
+  const id = req.params.id
+  const course = await Course.findByPk(id)
+  const user = await User.findByPk(id)
```

Comment:
I think that sequalize has a findAll method instead of findByPk, so that when you query users in a specific course it will return  all the users sort of like User.findAll({attributes: [userId] where: { courseId: ${id}}})